src/script/components/app-modal.ts

```diff
@@ -151,7 +151,7 @@ export class AppModal extends LitElement {
     if (this.open) {
       return html`
         <div id="background">
-          <div id="modal">
+          <div part="modal-container" id="modal">
```

Comment:
maybe an even more generic name?
Like `layout`? actually something we should discuss.
One of the things I've started to appreciate about the fast-components is that they have a very consistent part and css variable naming structure, thinking we could take their cue on this

Reply:
Im down with layout, I kinda would like it to still be "scoped" to the modal in a way, so maybe modal-layout?

Reply:
Changed to modal-layout (: